.github/workflows/release.yml

```diff
@@ -0,0 +1,91 @@
+name: "Release"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+  pull-requests: write
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
@ignacio-penas Is this for future possible change?

Reply:
@konstan I don't think I understand your question.

The trigger for the release action should be done only when merging into main so that the release PR is created against main too.

Reply:
ATM the git project uses `master` as the trunk branch. My question was regarding `main` branch name in the list of the branches (where indeed `master` is present).